packages/schematics/angular/application/index.ts

```diff
@@ -223,6 +223,36 @@ function addAppToWorkspaceFile(options: ApplicationOptions, workspace: Workspace
     // }
 
     workspace.projects[options.name] = project;
+
+    // tslint:disable-next-line:no-any
+    const schematics: any = {};
+
+    if (options.inlineTemplate !== undefined
```

Comment:
none of these should ever be undefined since the schema contains default values for all the options referenced here.

Reply:
Are you sure? I supposed `options` was the ones passed in the cli command, as the types are `x | undefined`. If so, it's just a quick fix, I will correct the PR later today.

Reply:
Fixed.